src/app/store/services/users.service.ts

```diff
@@ -106,14 +106,14 @@ export class UsersService {
   }
 
   signOut() {
-    this.router.navigateByUrl('/signin');
     localStorage.removeItem('token_expiration');
     localStorage.removeItem('user_key');
     localStorage.removeItem('ctemplar_mail');
     sessionStorage.removeItem('ctemplar_mail');
     localStorage.removeItem(PROMO_CODE_KEY);
     localStorage.removeItem(REMEMBER_ME);
     localStorage.removeItem(NOT_FIRST_LOGIN);
+    this.router.navigateByUrl('/signin');
```

Comment:
Great, I guess this solves the unable to find account details on expired tokens!